tests/urdf_examples/r2d2.urdf

```diff
@@ -0,0 +1,262 @@
+<robot name="flexible">
```

Comment:
If you copied a urdf from somewhere you should properly attribute the owner and check that their licence permits this copy

Also what purpose does this urdf serve? It is not called from anywhere